packages/components/src/LightBox/LightBox.tsx

```diff
@@ -92,4 +138,86 @@ export function LightBox({
   function handleMoveNext() {
     setCurrentImageIndex((currentImageIndex + 1) % images.length);
   }
+
+  function handleImageWrapperClick(
+    event: React.MouseEvent<HTMLDivElement, MouseEvent>,
+  ) {
+    const target = event.target as HTMLDivElement;
+    if (target.id === "imageWrapper") {
+      onRequestClose({ lastPosition: currentImageIndex });
+    }
+  }
+
+  function handleOnDragEnd(
+    event: MouseEvent | TouchEvent | PointerEvent,
+    { offset, velocity }: PanInfo,
+  ) {
+    const swipe = swipePower(offset.x, velocity.x);
+
+    if (swipe < -swipeConfidenceThreshold) {
+      handleMovePrevious();
+    } else if (swipe > swipeConfidenceThreshold) {
+      handleMoveNext();
+    }
+  }
+}
+
+interface ChildrenProps {
+  children: ReactNode;
+}
+
+function Title({ children }: ChildrenProps) {
+  return <span className={styles.title}>{children}</span>;
+}
+
+function Toolbar({ children }: ChildrenProps) {
+  return <div className={styles.toolbar}>{children}</div>;
+}
```

Comment:
I think we don't need to do this since it's just aliasing an element anyway. Save us a component runaround. Same goes for the below as well. What do you think?

Reply:
I did it to make it easier to read and understand the main component, but I'm fine with just having the divs directly there too